packages/tempus/src/Tempus_IntegratorAdjointSensitivity_decl.hpp

```diff
@@ -76,6 +76,24 @@ class IntegratorAdjointSensitivity :
    *    <li> "Mass Matrix Is Identity" (default: false) Whether the mass matrix
    *         is the identity matrix, in which some computations can be skipped.
    * </ul>
+   *
+   * To support use-cases with explicitly computed adjoint operators, the
+   * constructor takes an additional model evaluator for computing the adjoint
+   * W/W_op.  It is assumed the operator returned by this model evaluator is
+   * the adjoint, and so will not be transposed.  It is also assumed this
+   * model evaluator accepts the same inArgs as the forward model, however it
+   * only requires supporting the adjoint W/W_op outArgs.
+   */
+  IntegratorAdjointSensitivity(
+    Teuchos::RCP<Teuchos::ParameterList>                pList,
+    const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& model,
+    const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& adjoint_model);
+
+  /*! \brief Version of the constructor taking a single model evaluator. */
+  /*!
+   * This version takes a single model evaluator for the case when the adjoint
```

Comment:
Is the first part of this comment still accurate?  It looks like the IntegratorAdjointSensitivity now takes 2 MEs, one for the forward model, one for the adjoint?

Reply:
I see there is another constructor that takes a single ME - maybe the comment is intended to go there?

Reply:
@ikalash I don't understand your comment.  The comments for the first constructor, which takes two ME's, describes what the 2nd ME is for.  The 2nd constructor, with a single ME, has the correct comment, I believe.

Reply:
You're right.  I got thrown off b/c github only shows you a snippet of the code.